astro/src/content/docs/customize/look-and-feel/upgrade.mdx

```diff
@@ -20,6 +20,16 @@ New versions of FusionAuth sometimes include new or updated theme templates. If
 
 This page contains notes on changes in recent versions of FusionAuth that require changes to a customized theme.
 
+## Version 1.53.3
+
+Version `1.53.3` includes a change to persist the user's SSO decision through an external identity provider workflow. If the Google IdP's `loginMethod` is configured as `UsePopup` or `UseVendorJavaScript`, existing custom advanced themes require an update to the `helpers` template to incorporate the fix for the Google IdP. You can update the template via the API or by modifying the *Helpers* template in the admin UI.
+
+To allow the user's SSO decision to be persisted through a Google IdP login in an existing custom advanced theme, remove the `data-login_uri` attribute from the `div` with Id `g_id_onload` in the `googleButton` macro and add the following attribute in its place:
+
+```html
+data-callback="googleLoginCallback"
+```
```

Comment:
Should we have the before and after example? 


Replace

```html
data-login_uri="${currentBaseURL}/oauth2/callback?state=${idpRedirectState}&identityProviderId=${identityProvider.id}"
```
    
With
```html
data-callback="googleLoginCallback"
```

Reply:
I don't feel that strong about it - as long as we are confident we have given the user enough information for go find something in their template as it exists today, and to know what it should look like when it is done. 

I just know from personal experience if we leave any ambiguity, some customers will not be as confident as they could be otherwise. 

The current copy says:

> remove the `data-login_uri`

I know what that means, but as an end user that is stumbling through the theme, does that mean remove the key `data-login_uri=`,  the value `"${currentBaseURL}/oauth2/callback?state=${idpRedirectState}&identityProviderId=${identityProvider.id}"`, or both? 

We could also say, `remove the data-login_uri` attribute, and the value and replace with...  


Up to you, as long as we think it is easy to follow.